addons/website/static/src/systray_items/edit_website.xml

```diff
@@ -1,11 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 <t t-name="website.EditWebsiteSystray" owl="1">
-    <div class="o_edit_website_container d-none d-md-block" t-att-class="{ o_translatable: translatable }">
-        <a href="#" class="h-100" accesskey="a" t-on-click="startEdit" t-att-class="{'btn btn-primary d-flex align-items-center rounded-0 border-0 fw-normal': !translatable, 'o_nav_entry': translatable}">
-            <span t-if="translatable" class="fa fa-pencil me-2"/>
-            <span t-else="" class="fa me-2" t-att-class="{ 'fa-refresh fa-spin': this.state.isLoading, 'fa-pencil': !this.state.isLoading }"/>
-            <t t-esc="label"/>
+    <div class="o_menu_systray_item o_edit_website_container d-none d-md-block" t-att-class="{ o_translatable: translatable, o_not_translatable: !translatable }">
```

Comment:
Do we need the new `o_not_translatable` class? Seems like it is used for the CSS rule only, so why not `:not(.o_translatable)`? :thinking: I am not a fan of `:not(` in general but I prefer it over duplicating classes to do the `not` :wink:

Reply:
`o_translatable` is actually dead code.
I noticed that already when reviewing but :shrug: 
Only kept `o_not_translatable`, no need of `:not` as a bonus